hedera-mirror-rest/utils.js

```
@@ -983,11 +1002,55 @@ const loadPgRange = () => {
   pgRange.install(pg);
 };
 
+/**
+ * Checks that the timestamp filters contains either a valid range (greater than and less than filters that do not span beyond
+ * a configured limit), or at least one equals operator.
+ *
+ * @param timestampFilters an array of timestamp filters
+ */
+const maxTimestampRange = parseDuration(config.maxTimestampRange);
```

should parse the config in `config.js` and fail early if the provided value is invalid.

---

Changed to parse in config